coreos/provision.erb

```diff
@@ -14,7 +17,13 @@ coreos:
         [Unit]
         Description=Install coreos to disk
         [Service]
-        ExecStart=/bin/bash -c "/usr/bin/coreos-install -C  <%= @host.operatingsystem.release_name %> -d <%= @host.params['install-disk'] || '/dev/sda' %> -c /home/core/cloud-config.yml <% if @host.operatingsystem.major >= "557" -%>-b <%= @mediapath %><% end -%>  && wget -q -O /dev/null --no-check-certificate <%= foreman_url('built') %> && reboot"
+        Type=oneshot
+        ExecStart=/usr/bin/coreos-install \
+          -C <%= @host.operatingsystem.release_name %> \
+          -d <%= @host.params['install-disk'] || '/dev/sda' %> \
+          -c /home/core/cloud-config.yml <% if os_major >= 557 -%>-b <%= @mediapath %><% end %>
```

Comment:
@UnwashedMeme @mmoll is this really a Use Case to support such an old CoreOS version? [557.0](https://coreos.com/releases/#557.0.0) came out January 6, 2015 ...  with `Docker 1.3` + `etcd 0.4.6` personally I think we should just drop this part with ` <% if os_major >= 557 -%>` if somebody really wants to use such an old version he just have to remove the part `-b <%= @mediapath %>`.

I mean you use CoreOS because you want a minimal (always) up to date Linux Distro (also after the reboot CoreOS would fetch the newest version of the channel and install this version).